internal/dao/node.go

```diff
@@ -84,7 +86,7 @@ func (o DrainOptions) toDrainHelper(k kubernetes.Interface, w io.Writer) drain.H
 
 // Drain drains a node.
 func (n *Node) Drain(path string, opts DrainOptions, w io.Writer) error {
-	if err := n.ToggleCordon(path, true); err != nil {
+	if err := n.ToggleCordon(path, true); ignoreNodeAlreadyCordoned(err) != nil {
```

Comment:
I am wondering if we should actually have a call aka `ensureCordoned() bool` that we could just call here vs introducing a unilateral sentinel error?